pkgs/by-name/wi/wideriver/package.nix

```
@@ -0,0 +1,45 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib,
+  wayland,
+  wayland-scanner,
+  pkg-config,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "wideriver";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "alex-courtis";
+    repo = "wideriver";
+    rev = version;
+    sha256 = "sha256-Jn3DfflDlMSZMlSRBfzQeEUKgYLctd/PiQRWH8uwtlU=";
+  };
+
+  nativeBuildInputs = [
```

```suggestion
  buildInputs = [
```
See https://github.com/NixOS/nixpkgs/pull/311935#discussion_r1607278796

---

Reverted the change!
Will leave resolution open this time 😅